tests/generation/test_utils.py

```diff
@@ -4300,6 +4301,46 @@ def test_generate_from_inputs_embeds_with_bos_token_id_is_none(self):
         with self.assertRaises(ValueError):
             model.generate(max_length=20, bos_token_id=None)
 
+    @slow
+    def test_max_time(self):
```

Comment:
(this is almost a copy from the gpt2 test. the last assertion was incorrect, and it is fixed here)